Our OS provides superior way of dealing with updates (PackageKit).
Let's not bug the user with updates that are not yet packaged.

Lubomir Rintel <lkundrak@v3.sk>

diff -up VirtualBox-2.2.0_OSE/src/VBox/Frontends/VirtualBox/src/VBoxUpdateDlg.cpp.noupdate VirtualBox-2.2.0_OSE/src/VBox/Frontends/VirtualBox/src/VBoxUpdateDlg.cpp
--- VirtualBox-2.2.0_OSE/src/VBox/Frontends/VirtualBox/src/VBoxUpdateDlg.cpp.noupdate	2009-05-03 11:04:39.000000000 +0200
+++ VirtualBox-2.2.0_OSE/src/VBox/Frontends/VirtualBox/src/VBoxUpdateDlg.cpp	2009-05-03 11:05:19.000000000 +0200
@@ -160,12 +160,11 @@ void VBoxUpdateData::decode (const QStri
             mDate = QDate::fromString (parser [1], Qt::ISODate);
         }
 
-        /* Incorrect values handles as 'once per day' mode.
+        /* Incorrect values handles as 'never' mode.
          * This is the default value if there is no such extra-data. */
         if (mIndex == -1 || !mDate.isValid())
         {
-            mIndex = 0;
-            mDate = QDate::currentDate();
+            mIndex = NeverCheck;
         }
     }
 }
